import_data/tasks.py

```diff
@@ -10,13 +11,69 @@
 import time
 from datetime import timedelta, datetime
 import requests
-from invoke import task
+from invoke import context, task
 from invoke.exceptions import Failure
 from utils.lock import FileLock
 
+cc_exec = concurrent.futures.ThreadPoolExecutor()
+
 logging.basicConfig(level=logging.INFO)
 
 
+class PrefixedStream:
+    """
+    Wrapper class around a stream adding a prefix to each writen line.
+    """
+    def __init__(self, src_stream, prefix):
+        self.prefix = prefix
+        self.src_stream = src_stream
+        self.buffer = ''
+
+    def write(self, data):
+        self.buffer += data
+        self.flush()
+
+    def flush(self):
+        lines = self.buffer.split('\n')
+
+        for line in lines[:-1]:
+            self.src_stream.write(
+                '\033[0;90m[{}]\033[0;0m {}\n'.format(self.prefix, line)
+            )
+
+        self.buffer = lines[-1]
+        self.src_stream.flush()
+
+
+class PrefixedContext(context.Context):
+    """
+    Wrapper class around a pyinvoke context adding a prefix to each lines
+    outputed by ctx.run(..).
+    """
+    def __init__(self, ctx, prefix):
+        self.__dict__ = ctx.__dict__.copy()
```

Comment:
This feels a bit hackish. Why not using the parent's [`__init__`](http://docs.pyinvoke.org/en/1.2/api/context.html#invoke.context.Context.__init__) ?
```suggestion
        super().__init__(ctx.config)
```

Reply:
Indeed it works, I think I had some troubles with a previous implementation which may have led me to think that there was no "copy constructor" for `invoke.config.Config`